samples/unit-tests/series-tilemap/coloraxis/demo.js

```diff
@@ -30,4 +30,23 @@ QUnit.test('Tilemap and ColorAxis', function (assert) {
         'ColorAxis.max should be the same as max value in points (#11644)'
     );
 
+    chart.series[0].update({
+        tileShape: 'circle'
+    });
+
+    var point = chart.series[0].points[1];
+    point.setState('hover');
+
+    assert.notEqual(
+        point.graphic.element.getAttribute('cx'),
+        "NaN",
```

Comment:
Consider to use 

```
assert.ok(
        Highcharts.isNaN(point.graphic.element.getAttribute('cx')),
```

Reply:
I prefer the original `assert.notEqual`. It is stricter...

Reply:
Good to know, thanks!